src/synopsys-action/synopsys-bridge.ts

```diff
@@ -18,13 +18,12 @@ export class SynopsysBridge {
     if (!synopsysBridgePath) {
       info('Synopsys Bridge path not found in configuration')
       info('Looking for synopsys bridge in default path')
-
       if (osName === 'darwin') {
         synopsysBridgePath = path.join(process.env['HOME'] as string, SYNOPSYS_BRIDGE_DEFAULT_PATH_MAC)
       } else if (osName === 'linux') {
         synopsysBridgePath = SYNOPSYS_BRIDGE_DEFAULT_PATH_LINUX
       } else if (osName === 'win32') {
-        synopsysBridgePath = SYNOPSYS_BRIDGE_DEFAULT_PATH_WINDOWS
+        synopsysBridgePath = (process.env['USERPROFILE'] as String).concat(SYNOPSYS_BRIDGE_DEFAULT_PATH_WINDOWS)
```

Comment:
Instead of concat, can we check to use path.join

Reply:
done.